Make_Tables/createdatabase.py

```diff
@@ -0,0 +1,46 @@
+import mysql.connector
+import os
+
+directory = os.getcwd()
+myinfo = open(directory+"/mysqlinfo.txt",'r')
+ldata = myinfo.readlines()
+
+mydb = mysql.connector.connect(
+        host = ldata[0][:-1],
+        user = ldata[1][:-1],
+        password = ldata[2]
+	)
+
+mycursor = mydb.cursor()
+
+try:
+	mycursor.execute("drop database sac_data")
+except:
+	print("Database does exist already")
```

Comment:
Fix the error message
database does not exist

Reply:
Done changes it and squashed the commits also.